modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/database/H2TreeClientIndex.java

```diff
@@ -82,6 +82,11 @@ private int calculateInlineSize(IndexColumn[] cols, int inlineSize, CacheConfigu
         // Do nothing.
     }
 
+    /** {@inheritDoc} */
+    @Override public long totalRowCount(IndexingQueryCacheFilter partsFilter) {
```

Comment:
It confuse me. What is happen when request for select will be started on not affinity node? How the zero can affect query plan?